mantidproject/mantid
# Fix makensis syntax in windows packaging script

### Description of work

#### Summary of work


The nightly pipelined failed when to create a windows package. This line [here](https://github.com/mantidproject/mantid/blob/dbfaa71272ca1e2377af454b41ff57ad5c0ea27a/installers/conda/win/create_package.sh#L219) is executed, but the paths that are passed to `cmd.exe` are incorrect.

This PR corrects the syntax of these paths by removing the escapes.

### To test:

I have run this in the `main_nightly_deployment_prototype` workspace on `isiscloudwin1-1`. You either go and look at the `.exe` file in that directory, or delete this file, ensure this branch is checked out, then run again.

Build package from branch: https://builds.mantidproject.org/job/build_packages_from_branch/693/
This is limited to windows due to the scope of the code changes.





---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
